corehq/pillows/mappings/user_mapping.py

```diff
@@ -95,7 +96,20 @@
                                        },
                                        'type': 'object'},
                 'status': {'type': 'string'},
-                'user_data': {'type': 'object', 'enabled': False},
+                'user_data_es': {
```

Comment:
just cause i was changing the way we store the data. i think we usually assume that the couch model maps directly to the ES model. so i would expect if fetching a couch user from ES that the `user_data` field would be a dictionary. if i overwrote that and made that an array i felt that would be confusing